js/message_test.js

```diff
@@ -1049,13 +1049,7 @@ describe('Message test suite', function() {
     var nested = new proto.jspb.test.Deeply.Nested.Message();
     nested.setCount(5);
     msg.setDeeplyNestedMessage(nested);
-
-    // After a serialization-deserialization round trip we should get back the
```

Comment:
Does it still have the same test coverage? I thought the original bug can only be revealed after a round-trip. Is it not the case?